### ISSUE TITLE
Backboa support for Amazon Cloud Drive

### ISSUE THREAD

**Issue by karudonaldson** [omega8cc/boa#808]
I was wondering if anyone has done anything to utilise Amazon Cloud Drive.
It seems like a decent price for unlimited storage so worthy of an option IMO.

The only issue I'd imagine is encrypting all of the data being sent, I guess this is the feature I'd be looking to implement if ACD were to be used.

Thoughts?

**Comment by omega8cc** [comment]
I don't think it is support in duplicity: https://bugs.launchpad.net/duplicity/+bug/1493632

**Comment by karudonaldson** [comment]
This is looking promising: 
https://entangledthought.wordpress.com/2015/11/06/duplicity-and-amazon-cloud-drive/
